### ISSUE TITLE
The `'` character is converted into `\"` in the title

### ISSUE THREAD

**Issue by matt6697** [davetcc/hugojoomla#19]
The `'` character is converted into `\"` in the title (** What's new** is converted to**What\"s new**)

**Comment by matt6697** [comment]
The issue is related with escaping the title string.

@davetcc : The **escapeit()** function seems to remove code blocks, hyperlinks, and linebreaks. need more investigation to identify which part of the function converts `'` into `\"`but why escaping the joomla article which is very unlikely to include linebreaks, code blocks, and hyperlinks ? (is it ever possible to add such html code in the title of a joomla article ?)

```
public String getTitle() {
        return escapeIt(title);
    }
```

**Comment by davetcc** [comment]
I completely agree, it's highly unlikely the title would contain any such special characters, I probably just did it to be belt and braces basically during the conversions of my sites originally. But if it's causing problems now, then that should probably be re-evaluated.

**Comment by davetcc** [comment]
Let me take a look at the function now.

**Comment by davetcc** [comment]
I'm not sure how needed the escape it function is. It served two purposes originally that the markdown generation may have solved. It did a few character conversions to ensure valid files were always generated. But it also tried to convert code blocks in the code/pre tag form into triple back-tick form. 

One thing to bear in mind is that the title is in params section of the front matter, so any " and ' will really need to be escaped somehow to ensure validity of the document.
